docs/README.ios.md

```diff
@@ -0,0 +1,227 @@
+
+1. XCode 6.0.1 against iOS SDK 8.0 on 10.9 (Mavericks)
+2. XCode 6.1.0 against iOS SDK 8.1 on 10.10 (Yosemite)
+3. XCode 6.3.0 against iOS SDK 8.3 on 10.10 (Yosemite)
+4. Xcode 6.4.0 against iOS SDK 8.4 on 10.10 (Yosemite)
+5. Xcode 7.x against iOS SDK 9.x on 10.10 (Yosemite)
+6. Xcode 7.x against iOS SDK 9.x on 10.11 (El Capitan)
+7. Xcode 7.x against iOS SDK 9.x on 10.12 (Sierra)
+
+The preferred iOS SDK Version is 8.1.
+
+### 3.2 Install Kodi build depends
+
+Kodi requires a set of build dependencies to be built and installed before you
+will be able to build the Kodi main binary. These often just called *depends*
+are installed using the commands described below (with the latest iOS SDK found
+on your system).
+
+In order to speedup compilation it is recommended to use `make -j$(getconf
+_NPROCESSORS_ONLN)` instead of `make` to compile on all available processor
+cores. The build machine can also be configured to do this automatically by
+adding `export MAKEFLAGS="-j(getconf _NPROCESSORS_ONLN)"` to your shell config
+(e.g. `~/.bashrc`).
+
+#### 3.2.a Compiling as 32 bit armv7 libraries (recommended for most users)
+
+    cd $HOME/Kodi
+    cd tools/depends
+    ./bootstrap
+    ./configure --host=arm-apple-darwin
+    make
+
+#### 3.2.b Compiling as 64 bit arm64 libraries
+
+    cd $HOME/Kodi
+    cd tools/depends
+    ./bootstrap
+    ./configure --host=arm-apple-darwin --with-cpu=arm64
+    make
+
+#### 3.3.c Advanced topics
+
+The dependencies are built into `tools/depends` and installed into
+`/Users/Shared/xbmc-depends`.
+
+**ADVANCED developers only**: If you want to specify an iOS SDK version (if
+multiple versions are installed) - then append it to the configure line
+above. The example below would use the iOS SDK 8.0:
+
+    ./configure --host=arm-apple-darwin --with-sdk=8.0
+
+Ensure that you also adapt the Xcode project to use this SDK version.
+
+### 3.3 Compile Kodi binary addons
+
+Kodi maintains a set of binary addons (PVR clients, Visualizations, Audio DSP
+plugins and more). They can be built as shown below:
+
+    make -C target/binary-addons
+
+**NOTE**: If you only want to build specific addons you can specify like this:
+
+    make -C target/binary-addons ADDONS="pvr.hts pvr.dvblink"
+
+## 4 How to compile Kodi
+
+### 4.1 Using Xcode (or xcodebuild)
+
+#### 4.1.1 Generate CMake project files
+
+Before you can use Xode to build Kodi, the Xcode project has to be generated
+with CMake. Note that CMake is compiled as parts of the depends doesn't have
+to be installed separately. Also a Toolchain-file has been generated with is
+used to configure CMake.
+
+    mkdir $HOME/Kodi/build
+    cd $HOME/Kodi/build
+    /Users/Shared/xbmc-depends/buildtools-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake ../project/cmake
+
+The toolchain file location differs depending on your iOS and SDK version.
```

Comment:
please make it clear that the iphoneos9.3_armv7-target has to be replaced with iphoneos<SDKVERSION>_<CPU>-target ... i fear we get support questions here if its not in baby speech ;).